app/models/claim.rb

```diff
@@ -120,6 +124,10 @@ def can_claim_youth_court?
       plea_category.match(/category_[12]a/)
   end
 
+  def additional_fees_applicable?
+    [can_claim_youth_court?].excluding(false).present?
```

Comment:
`[can_claim_youth_court?].any?`

Reply:
yeah the intention was exactly for what @elken suggested - to future proof for future examples of "additional fee", the only reason I wrote it this way was so it would be easy to read if we had loads of different fees, but maybe it's not easier to read after all aha. Maybe for now I'll do away with the method and use can_claim_youth_court? for now and we can update once we get to that point to prevent redundant code

Reply:
actually thinking about it, I think it's still worth having the additional_fees_applicable? method so that we don't have to overhaul loadsa tests when we get to the point where we'd need to add another additional fee

Reply:
Just change `match` to `match?` if you want a bool, but `nil` is still falsey so I don't know if there's a good argument to doing so

Reply:
yeah you're right it's unnecessary - updated!